File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] arr) {
        Arrays.sort(arr);
        int count =1;
        int n = arr.length;
        int res = Integer.MIN_VALUE;
        for(int i=1;i<n;i++){

            if(arr[i]==arr[i-1]+1){
                count++;
            }
            else if(arr[i]==arr[i-1]){
                continue;
            }

            
            else{
                res = Math.max(res,count);
                count= 0;
                continue;
            }
            res = Math.max(res,count);


            
        }

        return res;
    }
}

File: Data Structures & Algorithms/pascals-triangle/submission-0.py
class Solution:
    def generate(self, n: int) -> List[List[int]]:
        res = []
        for i in range(n):
            row = [1]*(i+1)

            for j in range(1,i):
                row[j] = res[i-1][j]+res[i-1][j-1]
            
            res.append(row)
        
        return res